### PR TITLE
fix: Move search result observer to onCreateView

### LINKED ISSUE #2487: Move search result observer to onCreateView
https://github.com/fossasia/open-event-attendee-android/blob/99d61f40bab989c6e8de5770861c093cd710e7ea/app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt#L79-L83

### PR DESCRIPTION
**Pull request by refactor-droidyy**
Fixes #2487 Move search result observer to onCreateView

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt [hunk 1]
```diff
@@ -281,8 +282,12 @@ class SearchResultsFragment : Fragment(), CompoundButton.OnCheckedChangeListener
...
 
         rootView.searchText.addTextChangedListener(object : TextWatcher {
-            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) { /*Do Nothing*/ }
-            override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) { /*Do Nothing*/ }
+            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) { /*Do Nothing*/
+            }
+
+            override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) { /*Do Nothing*/
+            }
+
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt]: why did you make these changes?

Comment (refactor-droidyy) [reply app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt]: @nikit19 I just reformat the code for proper alignment nothing change in it except the search result observer. Is there  any change required?

Comment (nikit19) [reply app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt]: It was correct already

Comment (refactor-droidyy) [reply app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt]: @nikit19 so do I need to update the PR with above changes restored as the original one in SearchResultFragment.kt file keeping the search result observer same.

Comment (nikit19) [reply app/src/main/java/org/fossasia/openevent/general/search/SearchResultsFragment.kt]: Yes

**Pull request merged**